src/package/packageConvert.ts

```diff
@@ -140,14 +142,16 @@ export async function convertPackage(
  * @private
  */
 export async function createPackageVersionCreateRequest(
-  context: { installationkey?: string; definitionfile?: string; buildinstance?: string },
+  context: { installationkey?: string; definitionfile?: string; buildinstance?: string; seedmetadata?: string },
   packageId: string,
   apiVersion: string
 ): Promise<PackagingSObjects.Package2VersionCreateRequest> {
   const uniqueId = uniqid({ template: `${packageId}-%s` });
   const packageVersTmpRoot = path.join(os.tmpdir(), uniqueId);
   const packageVersMetadataFolder = path.join(packageVersTmpRoot, 'md-files');
+  const seedMetadataFolder = path.join(packageVersTmpRoot, 'seed-md-files');
   const packageVersBlobDirectory = path.join(packageVersTmpRoot, 'package-version-info');
+  const seedMetadataZipFile = path.join(packageVersBlobDirectory, 'seed-metadata-package.zip');
```

Comment:
seedMetadataZipFile is applicable to plain vanilla P2VC as well.  am bit confused since this code change is in packageConvert.ts

Reply:
never mind i see that it is added on packageVersionCreate.ts as well